iscsi/iscsi.go

```diff
@@ -30,6 +30,7 @@ var (
 	sleep              = time.Sleep
 )
 
+// iscsiSession contains information avout an iSCSI session
```

Comment:
This commit https://github.com/kubernetes-csi/csi-lib-iscsi/pull/29/commits/72717b863be93ebafc090787c8d9207820d421a4 changes can be part of another commits which changed these functions or used these functions.